nifi-docker/dockerhub/Dockerfile

```diff
@@ -23,11 +23,15 @@ LABEL site="https://nifi.apache.org"
 ARG UID=1000
 ARG GID=1000
 ARG NIFI_VERSION=1.8.0
-ARG MIRROR=https://archive.apache.org/dist
+ARG MIRROR_BASE_URL=https://archive.apache.org/dist
+ARG ARCHIVE_BASE_URL=https://archive.apache.org/dist
+
+ENV NIFI_BASE_DIR=/opt/nifi
+ENV NIFI_HOME=${NIFI_BASE_DIR}/nifi-${NIFI_VERSION}
+
+ENV NIFI_BINARY_PATH=${NIFI_BINARY_PATH:-/nifi/${NIFI_VERSION}/nifi-${NIFI_VERSION}-bin.zip}
```

Comment:
Did you forget to add `/opt` here to this? `PATH:-/nifi/${NIFI_VERSION}`?

From the diff, it doesn't look like `/nifi` would be a valid starting path.

Reply:
it's the path behind the domain in the url, not on the filesystem